boards/arm/stm32/b-g474e-dpow1/README.txt

```diff
@@ -52,7 +52,7 @@ Development Environment
   * Build NuttX.
 
   * Flash the code using:
-    $ openocd -f interface/stlink.cfg -f target/stm32g4x.cfg -c init \
+    $ sudo openocd -f interface/stlink.cfg -f target/stm32g4x.cfg -c init \
```

Comment:
IMO such things like sudo should not be stated as it is very specific and user may run the system with root permissions already. But if that is convenient for copy pasting then I'm fine with that. I mean that some instructions state `apt install xxx` and other `sudo apt install xxx`